src/docs/feature-flags.mdx

```diff
@@ -227,9 +230,10 @@ class TestFeatureFeatureHandler(features.FeatureHandler, PercentageRollout):
 
 ## After launch
 
-After your feature has been mainlined and is available for all hosted customers.
-You should go back and enable the feature in `conf/server.py` which will enable
-the feature for on-premise users.
+After your feature has been mainlined and is available for all hosted customers:
```

Comment:
This is the main change, rest is mostly the formatter and some grammatical fixes.